src/support/z_session.erl

```diff
@@ -207,7 +207,7 @@ add_script(Script, ContextOrPid) ->
     z_transport:session(javascript, Script, ContextOrPid).
 
 %% @doc Split the scripts from the context and add the scripts to the session pages.
--spec add_script(#context{}) -> ok.
+-spec add_script(#context{}) -> #context{}.
```

Comment:
The spec in line 210 uses a record, please define a type for the record and use that instead.